packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -240,8 +257,7 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         Clock nextClock = LibClock.wrap(nextDuration, Timestamp.wrap(uint64(block.timestamp)));
 
         // INVARIANT: There cannot be multiple identical claims with identical moves on the same challengeIndex. Multiple
-        // claims
-        //            at the same position may dispute the same challengeIndex. However, the must have different values.
+        // claims at the same position may dispute the same challengeIndex. However, they must have different values.
```

Comment:
Styling nit
```suggestion
        //            claims at the same position may dispute the same challengeIndex. However, they must have
        //            different values.
```

Reply:
This'll make forge fmt mad due to line length, doing locally :)